pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -267,7 +267,10 @@ func parseQuery(model *simplejson.Json) (*CloudWatchQuery, error) {
 		period = int(d.Seconds())
 	}
 
-	alias := model.Get("alias").MustString("{{metric}}_{{stat}}")
+	alias := model.Get("alias").MustString()
```

Comment:
I think this will do the same?

https://github.com/grafana/grafana/blob/master/pkg/components/simplejson/simplejson.go#L311

Reply:
It could be empty string if user set `Alias` field, and then remove all string.